src/main/kotlin/de/bund/digitalservice/useid/statics/DeeplinkController.kt

```diff
@@ -0,0 +1,38 @@
+package de.bund.digitalservice.useid.statics
+
+import mu.KotlinLogging
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+import reactor.core.publisher.Mono
+
+@Controller
+class DeeplinkController {
+
+    private val log = KotlinLogging.logger {}
+
+    @GetMapping(
+        path = [".well-known/apple-app-site-association"],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    fun getIOSUniversalLink(): Mono<ResponseEntity<IOSUniversalLink>> {
+        val appIds = listOf(
+            "VDTVKQ35RL.de.bund.digitalservice.UseID",
+            "VDTVKQ35RL.de.bund.digitalservice.UseID-Preview"
+        )
+        val components = listOf(Component(pathUrlName = "/eID-Client"))
+        val details = listOf(Details(appIds, components))
+        val universalLink = UniversalLink(details)
```

Comment:
What if something changes here? That would mean we have to manually modify this code here?

Reply:
yes, it's the trade-off we have to face :( it's not a great developer experience but I see:
1. no read file system
2. we don't change this configuration that often